stactools_spot/stactools/spot/stac_templates.py

```diff
@@ -0,0 +1,63 @@
+from datetime import datetime
+
+from pystac import (
+    Catalog,
+    Collection,
+    Extent,
+    Link,
+    Provider,
+    SpatialExtent,
+    TemporalExtent,
+)
+
+SPOT_SENSOR = {"S4": "SPOT 4", "S5": "SPOT 5"}
+
+GeobaseCatalog = Catalog(id="Geobase",
+                         description="STAC Catalog for Geobase",
+                         title=None,
+                         stac_extensions=None)
+
+SpotProviders = [
+    Provider(
+        "Government of Canada",
+        "Natural Resources Canada Centre for Topographic Information",
+        ["licensor", "processor"],
+        "www.geobase.ca",
+    ),
+    Provider("Sparkgeo", "info@sparkegeo.com", ["processor", "host"],
+             "www.sparkgeo.com"),
+    Provider("PCI Geomatics", "info@pci.com", ["processor", "host"],
+             "www.pcigeomatics.com"),
+]
+
+SpotExtents = Extent(
+    SpatialExtent([[0.0, 0.0, 0.0, 0.0]]),
```

Comment:
What's going on here?

Reply:
I've corrected the providers (Sparkgeo was providing a COG version in the repo I converted from) and included a proper spatial extent.